understanding/22/redundant-entry.html

```diff
@@ -72,6 +72,7 @@ <h2>Examples of Redundant Entry</h2>
       <ul>
          <li>A form requests the user’s corporate identification number (ID) in the first step of a process to purchase a new computer. In the 3rd step the user is asked to confirm that the computer will belong to the user (rather than a colleague), and re-shows the ID. It allows the user to change the ID, but defaults to the previously entered one.</li>
          <li>A form on an e-commerce website allows the user to confirm that the billing address and delivery address are the same address.</li>
+				 <li>A user submits a checkout form with an incorrect credit card number in it. The page updates, showing an error message, and doesn't clear any of the completed form fields.</li>
```

Comment:
```suggestion
				 <li>A user submits a checkout form with an incorrect credit card number in it. The page updates, showing an error message. Submitted information, such as credit card number, is not cleared from the form.</li>
```
Wondering if this is slightly simpler to read? No strong opinion.